neptune/exceptions.py

```diff
@@ -71,32 +112,132 @@ def __init__(self, expected_type, actual_type):
+
+
+class NeptuneMissingProjectQualifiedNameException(NeptuneException):
+    def __init__(self):
+        message = """
+{h1}
+----NeptuneMissingProjectQualifiedNameException-------------------------------------------------------------------------
+{end}
+Neptune client couldn't find your project name.
+
+There are two options two add it:
+    - specify it in your code 
+    - set an environment variable in your operating system.
+
+{h2}CODE{end}
+Pass it to {bold}neptune.init(){end} via {bold}project_qualified_name{end} argument:
+    {python}neptune.init(api_token='YOUR_LONG_TOKEN', project_qualified_name='WORKSPACE_NAME/PROJECT_NAME'){end}
+
+{h2}ENVIRONMENT VARIABLE{end}
+or export or set an environment variable depending on your operating system: 
+
+    {correct}Linux/Unix{end}
+    In your terminal run:
+       {bash}export {env_project}=WORKSPACE_NAME/PROJECT_NAME{end}
 
+    {correct}Windows{end}
+    In your CMD run:
+       {bash}set {env_project}=WORKSPACE_NAME/PROJECT_NAME{end}
 
-class IncorrectProjectQualifiedName(NeptuneException):
+and skip the {bold}project_qualified_name{end} argument of {bold}neptune.init(){end}: 
+    {python}neptune.init(api_token='YOUR_LONG_TOKEN'){end}
+
+You may also want to check the following docs pages:
+    - https://docs.neptune.ai/workspace-project-and-user-management/index.html
+    - https://docs.neptune.ai/getting-started/quick-starts/log_first_experiment.html
+    
+{correct}Need help?{end}-> https://docs.neptune.ai/getting-started/getting-help.html
+""".format(**{'env_project': envs.PROJECT_ENV_NAME, **STYLES})
+        super(NeptuneMissingProjectQualifiedNameException, self).__init__(message)
+
+
+class NeptuneIncorrectProjectQualifiedNameException(NeptuneException):
     def __init__(self, project_qualified_name):
-        super(IncorrectProjectQualifiedName, self).__init__('Incorrect project qualified name "{}". '
-                                                            'Should be in format "namespace/project_name".'
-                                                            .format(project_qualified_name))
+        message = """
+{h1}
+----NeptuneIncorrectProjectQualifiedNameException-----------------------------------------------------------------------
+{end}
+Project qualified name {fail}{project_qualified_name}{end} you specified was incorrect.
+
+The correct project qualified name should look like this {correct}WORKSPACE/PROJECT_NAME{end}.
+It has two parts:
+    - {correct}WORKSPACE{end}: which can be your username or your organization name
+    - {correct}PROJECT_NAME{end}: which is the actual project name you chose 
+
+For example, a project {correct}jakub-czakon/blog-hpo{end} parts are:
+    - {correct}jakub-czakon{end}: a {underline}WORKSPACE{end} my username
+    - {correct}blog-hpo{end}: a {underline}PROJECT_NAME{end} a project name
+    
+The URL to this project looks like this: https://ui.neptune.ai/jakub-czakon/blog-hpo 
```

Comment:
Just a quick reminder about our on-prem deployments (in their cases the link will be different) - but I'm ok with that if you are.

Also, I don't have a very strong opinion on using your particular username, but keep in mind that `jakub-czakon` might not be an obvious `firstname-lastname` situation for everyone.

Reply:
Good point, changed to neptune-ai/credit-default-prediction